src_controller/controller.gd

```diff
@@ -27,6 +27,15 @@ var preview_alignment := "left"
 ## FUNCTIONS  #################################################
 
 func _ready() -> void:
+	# Drag and drop for choosing screensaver and loading script
+	get_window().connect("files_dropped", func(files: PackedStringArray):
+		match files[0].get_extension():
+			"png", "jpg", "webp", "jpeg":
```

Comment:
```suggestion
		match files[0].get_extension().to_lower():
			"png", "jpg", "webp", "jpeg", "svg", "svgz":
```
Some file extensions like jpg are sometimes capitalized and would return JPG, using to_lower() makes it more error-proof.
svg and svgz are also supported, people could use this to display their logo or something on screen so I added these as well.